### ISSUE TITLE
Training on raccoon dataset

### ISSUE THREAD

**Issue by fabiomazzarella** [experiencor/keras-yolo2#318]
Hi,
Many thanks for this wonderful project.

I am trying to train the network on the raccoon dataset and I am using the config.json as follows:

    "model" : {
        "backend":              "Full Yolo",
        "input_size":           416,
        "anchors":              [4.03,5.23, 5.60,9.47, 7.53,11.54, 9.92,8.75, 11.02,11.99],
        "max_box_per_image":    10,        
        "labels":               ["raccoon"]
    },

    "train": {
        "train_image_folder":   "raccoon_dataset/images/",
        "train_annot_folder":   "raccoon_dataset/annotations/",     
          
        "train_times":          2,
        "pretrained_weights":   "",
        "batch_size":           16,
        "learning_rate":        1e-4,
        "nb_epochs":            10,
        "warmup_epochs":        1,

        "object_scale":         5.0 ,
        "no_object_scale":      1.0,
        "coord_scale":          1.0,
        "class_scale":          1.0,

        "saved_weights_name":   "raccoon_full_yolo.h5",
        "debug":                false
    },

    "valid": {
        "valid_image_folder":   "",
        "valid_annot_folder":   "",

        "valid_times":          1
    }

where the anchors are those generated by gen_anchors.py. When I try a prediction on a test image, I always get no boxes detected. 

Is there anything I'm missing. Should I keep training on a larger number of epochs?

Thank you in advance!

**Comment by sherlocked777** [comment]
Did you solved the issue??
